Cubical/Data/Int/MoreInts/QuoInt/Properties.agda

```diff
@@ -237,81 +237,82 @@ sign-pos-suc-· m (signed s (suc n)) = refl
-  ℤ→Int (pos zero + m) ∎
-ℤ→Int+Int≡+ (pos (suc n)) m =
-  (ℤ→Int (pos (suc n))) +Int (ℤ→Int m)
-  ≡⟨ sym (sucInt+ (Int.pos n) (ℤ→Int m)) ⟩
-  sucInt ((Int.pos n) +Int (ℤ→Int m))
-  ≡⟨ cong sucInt (ℤ→Int+Int≡+ (pos n) m) ⟩
-  sucInt (ℤ→Int ((pos n) + m))
-  ≡⟨ sucℤ→Int ((pos n) + m) ⟩
-  ℤ→Int (sucℤ ((pos n) + m))
-  ≡⟨ cong ℤ→Int (sucℤ-+ˡ (pos n) m) ⟩
-  ℤ→Int ((pos (suc n)) + m) ∎
-ℤ→Int+Int≡+ (neg (suc zero)) m =
-  negsuc zero +Int ℤ→Int m
-  ≡⟨ sym (predInt+ (Int.pos zero) (ℤ→Int m)) ⟩
-  predInt ((Int.pos zero) +Int (ℤ→Int m))
-  ≡⟨ cong predInt (ℤ→Int+Int≡+ (neg zero) m) ⟩
-  predInt (ℤ→Int ((neg zero) + m))
-  ≡⟨ predℤ→Int ((neg zero) + m) ⟩
-  ℤ→Int (predℤ ((neg zero) + m))
-  ≡⟨ cong ℤ→Int (predℤ-+ˡ (neg zero) m) ⟩
-  ℤ→Int ((neg (suc zero)) +  m) ∎
-ℤ→Int+Int≡+ (neg (suc (suc n))) m =
-  negsuc (suc n) +Int ℤ→Int m
-  ≡⟨ sym (predInt+ (negsuc n) (ℤ→Int m)) ⟩
-  predInt ((negsuc n) +Int ℤ→Int m)
-  ≡⟨ cong predInt (ℤ→Int+Int≡+ (neg (suc n)) m) ⟩
-  predInt (ℤ→Int ((neg (suc n)) + m))
-  ≡⟨ predℤ→Int ((neg (suc n)) + m)⟩
-  ℤ→Int (predℤ ((neg (suc n)) + m))
-  ≡⟨ cong ℤ→Int (predℤ-+ˡ (neg (suc n)) m) ⟩
-  ℤ→Int ((neg (suc (suc n))) + m) ∎
-  -- uh, this looks like an extremely generic thing that could be done when defining any function from ℤ, thus only requiring defining it on `pos n` and `neg (suc n)` as one would kinda expect for integers; why don't we do that everywhere?`
-ℤ→Int+Int≡+ (neg zero) m = subst (λ n →  (ℤ→Int n) +Int (ℤ→Int m) ≡ ℤ→Int (n + m)) posneg (ℤ→Int+Int≡+ (pos zero) m)
-ℤ→Int+Int≡+ (posneg i) m = subst-filler (λ n →  (ℤ→Int n) +Int (ℤ→Int m) ≡ ℤ→Int (n + m)) posneg (ℤ→Int+Int≡+ (pos zero) m) i
-
-+'≡+ : _+'_ ≡ _+_
-+'≡+ = _+'_
-  ≡⟨ cong ( _∘_ (λ f → Int→ℤ ∘ f)) (funExt₂ ℤ→Int+Int≡+) ⟩
-  (λ n → (λ m → (Int→ℤ (ℤ→Int (n + m)))))
-  ≡⟨ funExt₂ (λ n m → (Iso.rightInv isoIntℤ (n + m))) ⟩
-  _+_ ∎
-
-op≡Intℤ : (Int → Int → Int) ≡ (ℤ → ℤ → ℤ)
-op≡Intℤ = isoToPath opIsoIntℤ
-
-+Int≡+' : (λ i → op≡Intℤ i) [ _+Int_ ≡ _+'_ ]
-+Int≡+' = transport-filler op≡Intℤ _+Int_
-
-+Int≡+ : (λ i → (op≡Intℤ ∙ refl) i) [ _+Int_ ≡ _+_ ]
+private
+  _+'_ : ℤ → ℤ → ℤ
+  _+'_ = Iso.fun (binaryOpIso isoIntℤ) Int._+_
+
+  sucℤ→Int : ∀ (n : ℤ) → Int.sucℤ (ℤ→Int n) ≡ ℤ→Int (sucℤ n)
+  sucℤ→Int (pos n) = refl
+  sucℤ→Int (neg zero) = refl
+  sucℤ→Int (neg (suc zero)) = refl
+  sucℤ→Int (neg (suc (suc n))) = refl
+  sucℤ→Int (posneg i) = refl
+
+  predℤ→Int : ∀ (n : ℤ) → Int.predℤ (ℤ→Int n) ≡ ℤ→Int (predℤ n)
+  predℤ→Int (pos zero) = refl
+  predℤ→Int (pos (suc n)) = refl
+  predℤ→Int (neg zero) = refl
+  predℤ→Int (neg (suc n)) = refl
+  predℤ→Int (posneg i) = refl
+
+  ℤ→Int+Int≡+ : ∀ (n m : ℤ) → (ℤ→Int n) Int.+ (ℤ→Int m) ≡ ℤ→Int (n + m)
+  ℤ→Int+Int≡+ n m = (ℤelim (λ n → ∀ (m : ℤ) → (ℤ→Int n) Int.+ (ℤ→Int m) ≡ ℤ→Int (n + m)) posℤ→Int+Int≡+ negsucℤ→Int+Int≡+ n) m
+    where
+    posℤ→Int+Int≡+ : ∀ (n : ℕ) (m : ℤ) → (ℤ→Int (pos n)) Int.+ (ℤ→Int m) ≡ ℤ→Int ((pos n) + m)
+    posℤ→Int+Int≡+ zero m =
+      (ℤ→Int (pos zero)) Int.+ (ℤ→Int m) ≡⟨
```

Comment:
That's too funky of a notation wrt. the rest of the library. I feel better with seperate lines for `≡` steps, with differing indentation. Also, it's not consistent with +'≡+

Reply:
Better now? Or is this not what you had in mind?

I also fixed the other series of equalities, I completely forgot it was there before. ^^"